### DISCUSSION TITLE
Создание кастомных строительных сетов

### DISCUSSION THREAD

**Discussion by KukushkaSex** [Sokol-Games-GmbH/PixelCombats.PublicApi#88] [Ideas]
`BuildBlocksSet.Add(тут описываем все айди блоков)`

**Comment by kkohno** [comment]
Так а у нас разве такого нет в АПИ? Нужно посмотреть... возможно сделаем побыстрому

**Comment by Kukushechka7-ezx** [comment]
Такого нету
